universal-application-tool-0.0.1/app/services/program/ProgramDefinition.java

```diff
@@ -47,12 +50,77 @@ public static Builder builder() {
   /** A human readable description of a Program, localized for each supported locale. */
   public abstract LocalizedStrings localizedDescription();
 
-  /** The list of {@link BlockDefinition}s that make up the program. */
```

Comment:
Question (blocking for an answer): Did you mean to remove this docstring?

Reply:
nope! thanks for catching!!!!